client/my-sites/plan-features/plan/docs/example.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import PlanHeader from '../header';
+
+export default React.createClass( {
+
+	displayName: 'Plan',
+
+	mixins: [ PureRenderMixin ],
+
+	render() {
+		const price = {
+			currencySymbol: '$',
+			dollars: '8',
+			cents: '25'
+		};
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/app-components/plan">Plan</a>
+				</h2>
+				<div>
+					<PlanHeader
+						popular
+						current
+						title={ 'Premium' }
+						planType={ 'premium' }
+						price={ price }
+						billingTimeFrame={ 'per month, billed yearly' }
+					/>
+				</div>
+			</div>
+		);
+	}
+} );
```

Comment:
Looks like github doesn't show line 43. You'll usually see a message in the diff if the file is missing one
![no-newline](https://cloud.githubusercontent.com/assets/1270189/15909290/7932bcda-2d7a-11e6-9aea-bf10995aedf5.png)